persistence/src/test/scala/hmda/persistence/processing/HmdaFilingSpec.scala

```diff
@@ -22,19 +23,19 @@ class HmdaFilingSpec extends ActorSpec with LarGenerators {
   "HMDA Filing" must {
     "Store 10 lars" in {
       for (lar <- lars10) {
-        probe.send(hmdaFiling, LarValidated(lar, ""))
+        probe.send(hmdaFiling, LarValidated(lar, SubmissionId("12345", period, 0)))
       }
       probe.send(hmdaFiling, GetState)
-      probe.expectMsg(HmdaFilingState(10))
+      probe.expectMsg(HmdaFilingState(Map(s"12345-$period-0" -> 10)))
     }
-    "read validated lars from HmdaFileValidator and save them" in {
-      val submissionId = SubmissionId("12345", period, 1)
-      val validator = createHmdaFileValidator(system, submissionId)
-      for (lar <- lars10) {
-        probe.send(validator, lar)
+    "Store additional 20 lars" in {
+      for (lar <- lars20) {
+        probe.send(hmdaFiling, LarValidated(lar, SubmissionId("9999", period, 1)))
       }
       probe.send(hmdaFiling, GetState)
-      probe.expectMsg(HmdaFilingState(10))
+      probe.expectMsg(HmdaFilingState(Map(s"12345-2017-0" -> 10, s"9999-$period-1" -> 20)))
```

Comment:
Small comment: For consistency, you may want to change `"12345-2017-0"` to `"12345-$period-0"`, or update them all to use the literal string `2017` instead of interpolating.